services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core2/_workflow_runner.py

```diff
@@ -66,14 +64,14 @@ async def workflow_runner(
 
                 # fetching inputs from context
                 inputs: dict[str, Any] = {}
-                if step.input_types:
+                if step.input_types:  # type: ignore
```

Comment:
why these ignores?

Reply:
reverted. I don't have a solution for these typings for now